naas/configurations/default.py

```diff
@@ -0,0 +1,124 @@
+import os
+from naas.client import version
+from naas.configuration import Configuration
+from naas.logger import Logger
+
+
+class DefaultConfiguration:
+    """
+    This class is meant to store all default
+    configuration options that will be utilized
+    throughout the client.
+
+    These are all overrideable through the `configure`
+    interface
+    """
+    API_HOST = 'http://naas-api.deviceindependent.com'
```

Comment:
@arthuroe - This is a remnant that needs fixed in the ruby client, can we set this to be: `https://naas-api-staging.tinylab.com`

Reply:
Changed

Reply:
@muhallan - looks good! 👍